docs/examples/vault-support/rabbitmq-clusters.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: rabbitmq.com/v1beta1
+kind: RabbitmqCluster
+metadata:
+  name: cluster-vault-a
+  namespace: default
+  annotations:
+    rabbitmq.com/topology-allowed-namespaces: "*"
+spec:
+  image: rabbitmq:3.9.7-management
+  replicas: 1
+  service:
+    type: NodePort
+  rabbitmq:
+    additionalConfig: |
+      loopback_users = none
+  secretBackend:
+    vault:
+      role: messaging-topology-operator
```

Comment:
see comment further down in relation to separating role for the cluster operator and the messaging topology operator